docs/pages/enroll-resources/auto-discovery/kubernetes/google-cloud.mdx

```diff
@@ -396,6 +396,7 @@ ssh_service:
   enabled: off
 discovery_service:
   enabled: "yes"
+  discovery_group: "gke-myproject"
```

Comment:
Do we have any instructions telling the user to update `discovery_group` in this and other Teleport configuration examples?

Reply:
We have it in the reference page and some other configurations.
I can add it to everything.
The description we use in the reference page is:
```yaml
    # discovery_group is used to group discovered resources into different
    # sets. This is required when you have multiple Teleport Discovery services
    # running. It prevents discovered services from colliding in Teleport when
    # managing discovered resources.
    # If two Discovery Services match the same resources, they must be in the
    # same discovery group.
    # If two Discovery Services match different resources, they must be in
    # different discovery groups.
    discovery_group: "disc-group"
```

Reply:
👍 I would just make it explicit how to assign this value. Is this the name of a Google Cloud project where Kubernetes clusters you want to discover are running?

Reply:
It's an opaque value. Can be a random string.
When multiple DiscoveryServices are running, it is used to ensure resources are not deleted when reconciling the list of databases.

Reply:
👍 I hadn't realized when leaving this comment that there is a partial, `docs/pages/includes/discovery/discovery-group.mdx`, that provides this information already. I think we can remove the `discovery_group` comments and use this partial instead.

Reply:
Removed the inline comment in the `yaml` code blocks.
Added the partial where it was missing.